### DISCUSSION TITLE
Add a "send feedback" button in each addon in settings page

### DISCUSSION THREAD

**Discussion by sfederici** [ScratchAddons/ScratchAddons#2564] [Ideas]
Adding a "send feedback" button in each addon in the settings page could make easier to refer to the correct addon

**Comment by GrahamSH-LLK** [comment]
> Adding a "send feedback" button in each addon in the settings page could make easier to refer to the correct addon

Would a suitable alternative be a dropdown menu with all of the addons on the feedback page? I feel like that would just make the addon areas really crowded.

**Reply by sfederici** [reply]
I guess so

**Comment by sfederici** [comment]
I guess this can be closed now that each addon has its addon ID visible in the https://scratchaddons.com/addons/ page.

**Reply by BroJac5246** [reply]
Not everyone is aware of that page, though.

**Reply by WorldLanguages** [reply]
Discussions can't be closed, they can only be locked. I don't think this post deserves to be locked.
